src/bitmessagekivy/baseclass/common.py

```diff
@@ -0,0 +1,176 @@
+    return "99+" if total_msg > 99 else str(total_msg)
+
+
+def avatarImageFirstLetter(letter_string):
+    """This function is used to the first letter for the avatar image"""
+    try:
+        if letter_string[0].upper() >= 'A' and letter_string[0].upper() <= 'Z':
+            img_latter = letter_string[0].upper()
+        elif int(letter_string[0]) >= 0 and int(letter_string[0]) <= 9:
+            img_latter = letter_string[0]
+        else:
+            img_latter = '!'
+    except ValueError:
+        img_latter = '!'
+    return img_latter if img_latter else '!'
+
+
+def AddTimeWidget(time):  # pylint: disable=redefined-outer-name, W0201
+    """This method is used to create TimeWidget"""
+    action_time = TimeTagRightSampleWidget(
+        text=str(ShowTimeHistoy(time)),
+        font_style="Caption",
+        size=[120, 140] if platform == "android" else [64, 80],
+    )
+    action_time.font_size = "11sp"
+    return action_time
+
+
+def ShowTimeHistoy(act_time):
+    """This method is used to return the message sent or receive time"""
+    action_time = datetime.fromtimestamp(int(act_time))
+    crnt_date = datetime.now()
+    duration = crnt_date - action_time
+    display_data = (
+        action_time.strftime("%d/%m/%Y")
+        if duration.days >= 365
+        else action_time.strftime("%I:%M %p").lstrip("0")
+        if duration.days == 0 and crnt_date.strftime("%d/%m/%Y") == action_time.strftime("%d/%m/%Y")
+        else action_time.strftime("%d %b")
+    )
+    return display_data
+
+
+# pylint: disable=too-few-public-methods
+class AvatarSampleWidget(ILeftBody, Image):
+    """AvatarSampleWidget class for kivy Ui"""
+
+
+class TimeTagRightSampleWidget(IRightBodyTouch, MDLabel):
+    """TimeTagRightSampleWidget class for Ui"""
+
+
+class SwipeToDeleteItem(MDCardSwipe):
+    """Swipe delete class for App UI"""
+    text = StringProperty()
+    cla = Window.size[0] / 2
+    # cla = 800
+    swipe_distance = NumericProperty(cla)
+    opening_time = NumericProperty(0.5)
+
+
+class CutsomSwipeToDeleteItem(MDCardSwipe):
+    """Custom swipe delete class for App UI"""
+    text = StringProperty()
+    cla = Window.size[0] / 2
+    swipe_distance = NumericProperty(cla)
+    opening_time = NumericProperty(0.5)
+
+
+def empty_screen_label(label_str):
+    """This function show a default text when screen is empty."""
+    content = MDLabel(
+        font_style='Caption',
+        theme_text_color='Primary',
+        text=label_str,
+        halign='center',
+        size_hint_y=None,
+        valign='top')
+    return content
+
+
+def mdlist_message_content(queryreturn, data):
+    """Method to check the lenght of message content"""
+    for mail in queryreturn:
+        third_text = mail[3].replace('\n', ' ')
+        data.append({
+            'text': mail[1].strip(),
+            'secondary_text': mail[2][:10] + '...........' if len(
+                mail[2]) > 10 else mail[2] + '\n' + " " + (
```

Comment:
I would make `10` and `25` constants or variables

Reply:
I will use this function in other screens also, so i need to check the number of parameters in other screens also to make it more dynamic.